### ISSUE TITLE
Not working in React Native

### ISSUE THREAD

**Issue by stereodenis** [Payture/NodeJS-Payture-official#1]
There is no 'crypto' module in RN

**Comment by Payture** [comment]
Нужно попробовать добавить в package.json настройки
 "devDependencies": {
  "rn-nodeify": "^6.0.1"
}

"scripts": {
  "postinstall": "node_modules/.bin/rn-nodeify --install crypto --hack"
}
Проблема скорее всего связана с тем, что payture-official модуль опирается  на встроенные Node.js модули, которые  в  React Native это могут не работать без дополнительно установленных модулей.
В данном случае 'crypto' - это built-in Node.js модуль, а 'rn-nodeify'  - установит нужные связи между built-in Node.js  модулями для React Native

**Comment by stereodenis** [comment]
@Payture вот такая ошибка всплывает
```
Unable to resolve module `http` from `%APP_DIR%/node_modules/request/request.js`: Module does not exist in the module map
```

**Comment by Payture** [comment]
Добрый день!

Мы правильно понимаем, что ошибка всплывает в стороннем модуле request? Не
уверен, что этот вопрос к нам.

С уважением,
Игорь Коган
skype: koganigorpayture
payture.com

2017-10-03 16:05 GMT+03:00 Denis Evgrafov <notifications@github.com>:

**Comment by stereodenis** [comment]
@Payture у вас в коде используется request. Я так понимаю нам стоит сделать форк и переписать на поддерживаемую альтернативу, например superagent?

**Comment by fiery-spirit666** [comment]
Да, попробуйте заменить на альтернативный модуль. Модуль request используется только для простой отправки запросов (без каких-либо специальных настроек соединения) и получения ответов с сервера Payture.  По-сути, какой будет использоваться модуль  для отправки/приема http не критично.

С уважением,
Соловьева Елена
